tests/__snapshots__/test_help_messages.ambr

```diff
@@ -917,18 +917,45 @@
    Validates a deployed Snowflake Native App's setup script.                      
                                                                                   
   +- Options --------------------------------------------------------------------+
-  | --package-entity-id          TEXT  The ID of the package entity on which to  |
-  |                                    operate when definition_version is 2 or   |
-  |                                    higher.                                   |
-  | --app-entity-id              TEXT  The ID of the application entity on which |
-  |                                    to operate when definition_version is 2   |
-  |                                    or higher.                                |
-  | --project            -p      TEXT  Path where Snowflake project resides.     |
-  |                                    Defaults to current working directory.    |
-  | --env                        TEXT  String in format of key=value. Overrides  |
-  |                                    variables from env section used for       |
-  |                                    templates.                                |
-  | --help               -h            Show this message and exit.               |
+  | --interactive            --no-interactive          When enabled, this option |
```

Comment:
What does --interactive / --force mean for validate?

Reply:
For the scratch stage, we shouldn't really need interactive or force. So if that's the only reason to have them here, I'd be inclined to match the `app` command.

Reply:
updated